### ISSUE TITLE
[Attention] 超级视客营 MMEval 🥇🥈🥉

### ISSUE THREAD

**Issue by ice-tong** [open-mmlab/mmeval#54]
![image](https://user-images.githubusercontent.com/32220263/202389737-48b62f31-26c7-45a3-8c69-d14bb63ea7fe.png)

## 活动介绍
大家好，第一期 OpenMMLab 超级视客营实训活动开始啦！超级视客营实训活动提供十七个方向、上百个不同难度的任务供大家选择，不管你是初涉 AI 的新手还是资深炼丹师，都有适合你的任务供你选择。助力大家上手 OpenMMLab 开源算法库并参与项目建设。本期活动联合北京超级云计算中心，提供算力支持，为大家开发保驾护航。
活动参与方式：选择你感兴趣的任务，在 OpenMMLab 官网提交报名表。完成匹配后，即可和导师对接制定任务规划，开始上手开发。根据不同任务要求在对应的地址提交代码结果，出题方初步 review 通过后即可领取下一个任务或者坐等领奖。活动详情戳：[OpenMMLab 官网活动页](https://openmmlab.com/activity/codecamp)。

## 任务列表

任务题目 | 任务描述 | 技术标签 | 难易程度 | 积分数
-- | -- | -- | -- | --
Accuracy 支持更多深度学习框架（JAX） | 为 Accuracy 支持使用 jax.numpy.ndarray 计算，补充并通过单元测试 https://github.com/open-mmlab/mmeval/blob/main/mmeval/metrics/accuracy.py 多分派机制：https://mmeval.readthedocs.io/en/latest/design/multiple_dispatch.html | Python、 JAX | 新手任务 | 10
MeanIoU 支持更多深度学习框架 （JAX） | 为 MeanIoU 支持使用 jax.numpy.ndarray 计算，补充并通过单元测试 https://github.com/open-mmlab/mmeval/blob/main/mmeval/metrics/mean_iou.py#L21 多分派机制：https://mmeval.readthedocs.io/en/latest/design/multiple_dispatch.html | Python、JAX | 新手任务 | 10
为 MMEval 添加一个 Metric | 从列表中选择相关方向算法库中的一个 Metric，添加到 MMEval 中并对该算法库进行适配，详细任务列表见：https://github.com/open-mmlab/mmeval/issues/50 | Python | 进阶任务 | 30
为 MMEval 添加一个 Metric | 从列表中选择相关方向算法库中的一个 Metric，添加到 MMEval 中并对该算法库进行适配，详细任务列表见：https://github.com/open-mmlab/mmeval/issues/50 | Python | 进阶任务 | 30
为 MMEval 添加一个 Metric | 从列表中选择相关方向算法库中的一个 Metric，添加到 MMEval 中并对该算法库进行适配，详细任务列表见：https://github.com/open-mmlab/mmeval/issues/50 | Python | 进阶任务 | 30
为 MMEval 添加一个 Metric | 从列表中选择相关方向算法库中的一个 Metric，添加到 MMEval 中并对该算法库进行适配，详细任务列表见：https://github.com/open-mmlab/mmeval/issues/50 | Python | 进阶任务 | 30
为 MMEval 添加一个 Metric | 从列表中选择相关方向算法库中的一个 Metric，添加到 MMEval 中并对该算法库进行适配，详细任务列表见：https://github.com/open-mmlab/mmeval/issues/50 | Python | 进阶任务 | 30
为 MMEval 添加一个 Metric | 从列表中选择相关方向算法库中的一个 Metric，添加到 MMEval 中并对该算法库进行适配，详细任务列表见：https://github.com/open-mmlab/mmeval/issues/50 | Python | 进阶任务 | 30
为 MMEval 添加一个 Metric | 从列表中选择相关方向算法库中的一个 Metric，添加到 MMEval 中并对该算法库进行适配，详细任务列表见：https://github.com/open-mmlab/mmeval/issues/50 | Python | 进阶任务 | 30
为 MMEval 添加一个 Metric | 从列表中选择相关方向算法库中的一个 Metric，添加到 MMEval 中并对该算法库进行适配，详细任务列表见：https://github.com/open-mmlab/mmeval/issues/50 | Python | 进阶任务 | 30
为 MMEval 添加一个 Metric | 从列表中选择相关方向算法库中的一个 Metric，添加到 MMEval 中并对该算法库进行适配，详细任务列表见：https://github.com/open-mmlab/mmeval/issues/50 | Python | 进阶任务 | 30
为 MMEval 添加一个 Metric | 从列表中选择相关方向算法库中的一个 Metric，添加到 MMEval 中并对该算法库进行适配，详细任务列表见：https://github.com/open-mmlab/mmeval/issues/50 | Python | 进阶任务 | 30

活动报名地址：[报名表地址](https://openmmlab.com/activity/codecamp/apply)
根据任务难度可以获得对应积分，兑换不同奖品。另外完成任务后在知识社区发布学习心得即可获得额外积分（记得主动找小助手领取哦）。
活动交流群：群二维码
![image](https://user-images.githubusercontent.com/32220263/202391150-9eac5ecf-c5e3-40f2-b372-eb2d8d6a6ce9.png)

有任何疑问欢迎大家加入群聊或者 Issue 下参与讨论，快来完成挑战，加入 [OpenMMLab 贡献者队伍](https://openmmlab.com/contributor/path)吧～
